### PR TITLE
dopisane testy

### PR DESCRIPTION
**Pull request by adam-tokarski**
Dopisane testy do pozostałych funkcjonalności. 
Chociaż jest to nadmiarowe, pokryłem testami także kluczowe funkcje z utils.

pylint + pep8: https://gist.github.com/adam-tokarski/a413779ff041b2955777

### CODE REVIEW

File: src/presence_analyzer/tests.py [hunk 1]
```diff
@@ -86,6 +111,54 @@ def test_get_data(self):
...
+        self.assertEqual(resp.data, json.dumps(some_dict))
+        # if assigned mimetype is proper
+        self.assertEqual(resp.mimetype, u'application/json')
+
+    def test_seconds_since_midnight(self):
+        """
+        Test seconds counting
+        """
+        s_array = [(15, 8, 6), (12, 0, 0), (0, 0, 0), (23, 59, 59)]
+        secs_array = [(x[0] * 3600 + x[1] * 60 + x[2], utils.seconds_since_midnight(datetime.time(*x))) for x in s_array]
```
Comment (sargo) [inline src/presence_analyzer/tests.py]: Line too long. Try to cut it. Some examples you can find https://gist.github.com/krotkiewicz/3134c608c91cddbf21e5#file-list_comprehension-py

**Comment by adam-tokarski** [comment]
https://gist.github.com/adam-tokarski/e5012f4f00fdec158498

**Comment by sargo** [comment]
You mixed two tasks (tests and new tab) in one pull request. Tests are ok now (you fixed the one issue that I commented), but now there is much another comment in new added code. If you created two separate pull request then I could close one and you could work on second one. 

In this project it's not so important (only you develop it) but if team is bigger then it's important to share code earlier, because it's easier to merge and update by others.

**Comment by adam-tokarski** [comment]
Caching is removed, although I believed in that case (where data are immutable) it shouldn't be any problem. 

Yes, I see it now that it's not good way to mix in one pull requests many different tasks. I will avoid that in future.

**Pull request merged**